src/rider/main/kotlin/me/seclerp/rider/plugins/monogame/spritefont/SpriteFontXmlSchemaProvider.kt

```diff
@@ -0,0 +1,74 @@
+package me.seclerp.rider.plugins.monogame.spritefont
+
+import com.intellij.javaee.ExternalResourceManagerEx
+import com.intellij.openapi.module.Module
+import com.intellij.openapi.util.Key
+import com.intellij.openapi.util.ModificationTracker
+import com.intellij.openapi.vfs.VfsUtil
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiManager
+import com.intellij.psi.util.CachedValue
+import com.intellij.psi.util.CachedValueProvider
+import com.intellij.psi.util.CachedValuesManager
+import com.intellij.psi.xml.XmlFile
+import com.intellij.xml.XmlSchemaProvider
+import com.jetbrains.rd.platform.util.getLogger
+import com.jetbrains.rd.platform.util.getOrCreateUserData
+
+class SpriteFontXmlSchemaProvider : XmlSchemaProvider() {
+    companion object {
+        private val logger = getLogger<SpriteFontXmlSchemaProvider>()
```

Comment:
It's better to use `com.intellij.openapi.diagnostic.logger<T>`:
```suggestion
        private val logger = logger<SpriteFontXmlSchemaProvider>()
```

`getLogger<T>()` just calls `logger<T>()` under the hood. It exists for some Rd codebase historical reasons.